python-package/setup.py

```diff
@@ -46,41 +46,6 @@ def find_lib() -> List[str]:
     return LIB_PATH
 
 
-def copy_files(integrated_opencl: bool = False, use_gpu: bool = False) -> None:
```

Comment:
All of this copying-files-around stuff is unnecessary now the `setup.py` will only be invoked from the root of a directory containing all of the C/C++ source files + `CMake` scripts in the expected places.